.gitignore

```diff
@@ -4,3 +4,4 @@
 erl_crash.dump
 *.ez
 *.tar
+.tool-versions
```

Comment:
This is just for the version manager I use. If you don't want it in here, let me know and I'll get rid of it.